mu-plugins/central-hub/tests/phpunit/unit/meta-data/registerMetaBoxes.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Tests for the function register_meta_boxes().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Unit\Metadata;
+
+use Brain\Monkey;
+use function spiralWebDB\Metadata\register_meta_boxes;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_RegisterMetaBoxes
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @group   meta-data
+ */
+class Tests_RegisterMetaBoxes extends Test_Case {
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/meta-data/meta-box.php';
+	}
+
+	/*
+	 * Test register_meta_boxes() should register to add_action( 'admin_menu' ) when event fires.
+	 */
+	public function test_function_should_register_to_action_hook_when_event_fires() {
+		$this->assertTrue( has_action( 'admin_menu', 'spiralWebDB\Metadata\register_meta_boxes' ) );
+	}
+
+	/*
+	 * Test register_meta_boxes() will add a meta box for each store key that starts with 'metabox.'.
+	 */
+	function test_function_will_add_a_meta_box_for_each_store_key_that_starts_with_metabox() {
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_keys' )
+			->once()
+			->withNoArgs()
+			->andReturn( [ 'meta_box.events' ] );
+		Monkey\Functions\expect( 'KnowTheCode\ConfigStore\getConfigParameter' )
+			->once()
+			->with( 'meta_box.events', 'add_meta_box' );
+		Monkey\Functions\when( 'spiralWebDB\Metadata\add_meta_box' )
+			->justReturn( 'events' );
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_id' )
+			->once()
+			->with( 'meta_box.events' )
+			->andReturn( 'events' );
+
+		$this->assertTrue( register_meta_boxes() );
```

Comment:
@rgadon107 As I noted above, the function will not return `true`, as `true` is misleading and not valuable here. The assertions within the `except` above are the actual assertions for the test itself.

Now, PHPUnit will likely yell at you to have a separate assertSomething. You can do that with an `assertNull` instead, which is not valuable or useful; rather, it's just needed to make PHPUnit happy.

Reply:
@hellofromtonya 
>The assertions within the except above are _the actual assertions_ for the test itself.

Well, _that's interesting_. 

That's easy enough to change. 

I think you just gave me a _big clue_ as to how to set up the unit test for the next function in the `/src/meta-data/meta-box.php` file ( `render_meta_box()` ).